tests/test_taskgroups.py

```diff
@@ -582,7 +621,11 @@ async def child(fail):
 
 
 async def test_cancel_cascade():
+    tg2 = None
```

Comment:
Move this to just before the assertions (for consistency with other code).

Reply:
Done